docs/ranger/starting_with_ranger.md

```diff
@@ -0,0 +1,55 @@
+# Getting Started
+
+This guide will help you run your first Ranger job using a Solidity contract and a [CVL](/docs/cvl/index) [invariant](/docs/cvl/invariants).
+
+Ranger uses the same installation process, configuration files, and spec files as the [Certora Prover](/docs/user-guide/index). If you're already familiar with the Prover, getting started with Ranger will feel familiar.
```

Comment:
That Ranger uses CVL could be stated also `ranger_intro`?

Reply:
Done